public/worklets/voice-isolator-worklet.js

```
@@ -16,17 +15,17 @@ class VoiceIsolatorLegacyShim extends AudioWorkletProcessor {
       type: 'deprecated-worklet',
       message: 'voice-isolator-worklet.js is a legacy shim. Use /app/dsp-processor.js (processor name: dsp-processor).'
     });
+    this.port.postMessage({
+      type: 'error',
+      code: 'LEGACY_WORKLET',
+      message: 'voice-isolator-worklet.js is a deprecated pass-through shim. ' +
+               'Load /app/dsp-processor.js and register processor "dsp-processor" instead.'
+    });
   }
 
   process(inputs, outputs) {
```

ESLint

[error] 'inputs' is defined but never used.

(no-unused-vars)